Cargo.toml

```diff
@@ -19,23 +19,20 @@ resolver = "2"
 version = "0.12.0"
 authors = ["thin-edge.io team <info@thin-edge.io>"]
 edition = "2021"
-rust-version = "1.67"
+rust-version = "1.70"
```

Comment:
Beware that [bumping the rust version to 1.70 recently introduced issues](https://github.com/thin-edge/thin-edge.io/pull/2195), notably [in combination with zigbuild](https://github.com/thin-edge/thin-edge.io/pull/2196)

One needs to double check that the `just release` is still working.
- This is okay with my settings (zig=0.11, cargo-zigbuild= 0.17.1, rutsc=1.71.1)
- @reubenmiller can you confirm?

Reply:
It compiles fine, the issue was more with installing cargo-zigbuild which required rustc 1.70 or above.

Reply:
So just to be clear, bumping to 1.7.0 actually simplifies a lot, however I avoided bumping to `1.70.` in the [PR2195](https://github.com/thin-edge/thin-edge.io/pull/2195) just to avoid bumping the MSRV due to minor bumps in some package dependencies.